arviz/plots/backends/bokeh/autocorrplot.py

```diff
@@ -59,10 +59,16 @@ def plot_autocorr(
         title = Title()
         title.text = make_label(var_name, selection)
         ax.title = title
+        ax.x_range = DataRange1d(
+            start=0, end=max_lag, bounds=(0, len(np.arange(len(y)))), min_interval=5
```

Comment:
len(np.arange(len(y))) --> len(y)

Reply:
or actually `len(y) - 1`

Reply:
Yeah makes sense. I'll make the changes.

Also, what do you suggest the zoom in limit should be for these plots? I've set ```min_interval=5``` (float) for x-axis of ```autocorrplot```. This restricts zooming to a minimum interval of length 5.

Reply:
That sounds good

Reply:
Do you think it's suitable for other plots as well?

For examples, the y axes of ```traceplot``` and ```parallelplot``` are very detailed so I think they should be left unrestricted?

And x-axis of ```forestplot``` could use ```min_interval=1```?